xbmc/platform/linux/SysfsPath.cpp

```diff
@@ -0,0 +1,22 @@
+/*
+ *  Copyright (C) 2011-2018 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include "SysfsPath.h"
+
+bool CSysfsPath::Exists()
+{
+  std::ifstream file(m_path);
+
+  if (!file.is_open())
+  {
+    CLog::LogF(LOGERROR, "{} is not open for writing", m_path);
```

Comment:
```suggestion
    CLog::LogF(LOGERROR, "{} could not be opened for reading", m_path);
```

Also `Exists()` should not log an error if it doesn't find a file (because that's what the function is for, so it's not an error)